### DISCUSSION TITLE
Update to 2024.7.1

### DISCUSSION THREAD

**Discussion by DasCanard** [dani-garcia/vaultwarden#5163] [Ideas]
Would love to see a upgrade to at least Bitwarden Vault 2024.7.1 as they added the function do bulk assign items to collections.

**Comment by BlackDex** [comment]
I would love that too.
I'm working on it, but it takes some time testing everything really well unfortunately.

**Reply by Gerardv514** [reply]
Question, BW just released a Beta Chrome Extension which has a UI overhaul. Does that run off the WebVault version, or is that backend server work that needs to be updated? Currently I am not able to see the updated UI changes in that Beta app; I was surprised because I thought the UI update would have just been contained in the extension client.

**Reply by stefan0xC** [reply]
You would have to enable this feature flag (that was [added in the latest release](https://github.com/dani-garcia/vaultwarden/releases/tag/1.32.3)) if you want to test it: https://github.com/dani-garcia/vaultwarden/blob/2f20ad86f9283419845ff1b77c504c3dfd043578/.env.template#L350

**Reply by Gerardv514** [reply]
Ah OK. I checked/searched the wiki for instructions on how to enable "feature flags;" came up empty. Is there documentation on how to do this?

**Reply by stefan0xC** [reply]
I am not sure what you want to have explained? How to [configure settings](https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview) is explained in the wiki and the options are listed in the `.env.template` if you look at [the surrounding lines](https://github.com/dani-garcia/vaultwarden/blob/2f20ad86f9283419845ff1b77c504c3dfd043578/.env.template#L343-L352).

**Reply by Gerardv514** [reply]
Can I add this as an environment variable, which is the same as adding ADMIN_TOKEN      SIGNUPS_ALLOWED  false    PUSH_ENABLED  true


So... the environment variable for this is    EXTENSION-REFRESH    true            ?

**Reply by stefan0xC** [reply]
> How to [configure settings](https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview) is explained in the wiki

The setting is called 
https://github.com/dani-garcia/vaultwarden/blob/2f20ad86f9283419845ff1b77c504c3dfd043578/.env.template#L352

**Reply by Gerardv514** [reply]
Cool bro.. would of helped if fido2-vault-credentials wasn't apart of that line or something explaining to replace after the = sign with one of the above. 

OR

346 ## The following flags are available:   when using the EXPERIMENTAL_CLIENT_FEATURE variable

**Reply by stefan0xC** [reply]
Feel free to make a PR  if you think this would improve the documentation.

**Reply by Gerardv514** [reply]
Is there a wiki for that?

**Reply by stefan0xC** [reply]
Not yet but this project is hosted on GitHub so you can look [into its documentation for that](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/) (or create a new wiki page if you meant that).
